### ISSUE TITLE
Build base components

### ISSUE THREAD

**Issue by Hugo-Hbrt** [Hugo-Hbrt/stack-scroll#17]
Develop :

- Tag selector
- Tag
- Button
- Voter component
- Back to feed button
- Post card
- Comment
- Comment form
... 

Make sure each component is responsive, accessibility should be taken into account.
